### PR TITLE
New Game! Characters (Batch 1)

### PR DESCRIPTION
**Pull request by 1DavidCarbon**
The first batch of New Game! Characters are in. I will add in the rest of the characters later but the main ones are in.

### CODE REVIEW

File: waifus/Hifumi Takimoto.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+	"type" : 0,
+	"english-name" : "Hifumi Takimoto",
+	"image" : "https://vignette.wikia.nocookie.net/new-game/images/9/9d/Hifumi_Game_Render.png",
+	"japanese-name" : "滝本ひふみ",
+	"year" : ¯\_(ツ)_/¯,
```
Comment (yourWaifu) [inline waifus/Hifumi Takimoto.json]: If you already know that someone is legal, you can use the field ``definitely-legal`` and set that to true.

Comment (yourWaifu) [reply waifus/Hifumi Takimoto.json]: I added a quick fix for this. https://github.com/yourWaifu/is-your-waifu-legal/commit/17fa66d8785e35e9af4d0143933c95ab3bd3a335

File: waifus/Shizuku Hazuki.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+	"type" : 0,
+	"english-name" : "Shizuku Hazuki",
+	"image" : "https://vignette.wikia.nocookie.net/new-game/images/4/44/Th-c9.png",
+	"japanese-name" : "葉月しずく",
+	"year" : ¯\_(ツ)_/¯,
```
Comment (yourWaifu) [inline waifus/Shizuku Hazuki.json]: The ``¯\_(ツ)_/¯`` needs to be removed as that'll cause an error, if you don't know the year, then you can set it to null or not include the year.

File: CNAME [hunk 3]
```diff
@@ -0,0 +1 @@
+waifu_contributor.davidcarbon.ml
```
Comment (yourWaifu) [inline CNAME]: Umm, does this need to be here?

File: waifus/Hifumi Takimoto.json (hunk 1 above)
Comment (1DavidCarbon) [reply waifus/Hifumi Takimoto.json]: Added "definitely-legal" field

File: waifus/Shizuku Hazuki.json (hunk 2 above)
Comment (1DavidCarbon) [reply waifus/Shizuku Hazuki.json]: Added "definitely-legal" as substitution for the year

File: CNAME (hunk 3 above)
Comment (1DavidCarbon) [reply CNAME]: Nope

**Pull request merged**